add new line

1003Hisashi opened the pull request:
#What
エラーの出たファイルに対し改行追加

#Why
エラー解消

Commit:
add new line

エラーの出たファイルに対し追加

1003Hisashi commented:
![スクリーンショット 2019-12-28 20 37 35](https://user-images.githubusercontent.com/58619864/71543073-faafd900-29b1-11ea-9a05-ddf513742d70.png)

tenpadrummer commented on app/assets/stylesheets/config/colors.scss:
```diff
@@ -1,4 +1,4 @@
 $white: #fff;
```
scssの中には、まだ変数化できる色が残っています。
それらもすべて変数化してしまいましょう！

1003Hisashi replied:
複数回使用する色について変数化しました。
colors.scss
chat.scss
以上2つのファイルに変更を加えました。

tenpadrummer reviewed (commented):
レビュー依頼ありがとうございます！
haml、とても綺麗にかけていて素晴らしいです！
scssまわりで一点、コメントさせていただきました！
修正、がんばってくださいね！

Commit:
scss color

chat.scssファイルにおいて
複数回使用する色について変数化しました。
そのため、color.scssにも追加した箇所があります。ご確認をお願いいたします。

miyagit reviewed (approved):
LGTM
